### ISSUE TITLE
Graders handling supporting files

### ISSUE THREAD

**Issue by tansaku** [saasbook/rag#60]
The grader currently downloads every file in an assignments 'autograder' (or 'spec'?) folder.  However if a spec needs to load files relative to itself it has a problem in as much as the grader will try to load files relative to it's root, and so currently it's not clear how to allow 3rd parties to develop specs that require associated files.

**Comment by armandofox** [comment]
intent was that the initial fetch of the assignment files use http (which could include file:// urls for local case) so files could live anywhere, and then cache locally on /tmp or somewhere conveniennt

Armando Fox
Professor, Computer Science Division
Faculty Advisor, UC Berkeley MOOCLab
fox .at. cs .dot. berkeley .dot. edu
581 Soda Hall MC#1776, Berkeley, CA 94720-1776
+1.510.642.6820 / http://www.cs.berkeley.edu/~fox

TAKE MY CLASS FOR FREE VIA edX:  http://saas-class.org  
LIKE IT ON FACEBOOK:  http://facebook.com/saas-class
TRY MY BOOK (NOT FREE): http://www.saasbook.info (goes with the course)

**Comment by tansaku** [comment]
@armandofox yes, caching makes sense for efficiency, but we need a mechanism for instructors and other assignment developers to be able to fix problems in their specs etc.  At the moment a grader admin has to be involved in that process.  

This isn't an immediate blocker, but something that grader admins will have to be aware off; and something that will come up each time we make any change to an assignment spec.
